src/components/Counter/Counter.jsx

```diff
@@ -1,71 +1,102 @@
-import { useEffect, useState } from 'react';
-import styles from './Counter.css';
+import { useEffect, useState, useReducer } from 'react'
 
-const colors = {
-  yellow: 'rgb(236, 222, 153)',
-  green: 'rgb(52, 211, 153)',
-  red: 'rgb(239, 68, 68)',
+const yellow= 'rgb(236, 222, 153)'
+const initialCount = 0
+
+const countReducer = (state, action) => {
+  switch (action.type) {
+    case 'INCREMENT':
+      return state + 1
+
+    case 'DECREMENT':
+      return state - 1
+
+    case 'RESET':
+      return 0
+
+    default:
+      return state;
+  }
 };
 
+const colorReducer = (color, action) => {
+  switch (action.type) {
+    case 'COLOR':
+      return { color: action.payload.color }
+    default:
+      return color;
+  }
+};
 export default function Counter() {
-  const [count, setCount] = useState(0);
-  const [currentColor, setCurrentColor] = useState(colors.yellow);
+  // const [count, setCount] = useState(0)
+  // const [currentColor, setCurrentColor] = useState(pinkRGB)
+
+  const [currentColor, dispatch] = useReducer(colorReducer, yellow)
+  const [count, dispatchCount] = useReducer(countReducer, initialCount)
 
   useEffect(() => {
     if (count === 0) {
-      setCurrentColor(colors.yellow);
+      // setCurrentColor(pinkRGB)
+      dispatch({ type: 'COLOR', payload: { color: (yellow) } })
     }
 
     if (count > 0) {
-      setCurrentColor(colors.green);
+      // setCurrentColor(`rgb(52, 211, 153)`)
+      dispatch({ type: 'COLOR', payload: { color: `rgb(52, 211, 153)` } })
     }
 
     if (count < 0) {
-      setCurrentColor(colors.red);
+      // setCurrentColor(`rgb(239, 68, 68)`)
+      dispatch({ type: 'COLOR', payload: { color: `rgb(239, 68, 68)` } })
     }
-  }, [count]);
+  }, [count])
 
   const increment = () => {
-    setCount((prevState) => prevState + 1);
-  };
+    // setCount((prevState) => prevState + 1)
+    dispatchCount({ type: 'INCREMENT' })
+  }
 
   const decrement = () => {
-    setCount((prevState) => prevState - 1);
-  };
+    // setCount((prevState) => prevState - 1)
+    dispatchCount({ type: 'DECREMENT' })
+  }
 
   const reset = () => {
-    setCount(0);
-  };
+    // setCount(0)
+    dispatchCount({ type: 'RESET' })
+  }
 
   return (
-    <main className={styles.main}>
-      <h1 style={{ color: currentColor }}>{count}</h1>
-      <div>
+    <main className="bg-black bg-opacity-90 min-h-screen flex flex-col items-center justify-center text-4xl text-pink-500">
+      <h1 className="mb-5" style={{ color: currentColor.color }}>
+        {count}
+      </h1>
+      <div className="flex w-1/2 justify-around">
         <button
+          className="text-green-400 border-2 border-green-400 p-3"
```

Comment:
It looks like the colors you providing aren't rendering on the deployed site for the buttons.